helm-chart/renku-graph/values.yaml

```diff
@@ -25,6 +25,11 @@ global:
       # Password for the `tokenstorage` user
       # Generate one using: `openssl rand -hex 16`
       value:
+  triplesGenerator:
+    postgresPassword:
+      # Password for the `triplesgenerator` user
+      # Generate one using: `openssl rand -hex 16`
```

Comment:
I'm thinking if the `triplesGenerator` section shouldn't be indented two spaces more to be on the same level as `dbEventLog`, I mean under the `graph`. I think, the `tokenRepository` section that is just above the new `triplesGenerator` section may have wrong indentation, too.

Reply:
Thanks! Love that yaml stuff 😆